Source/AliveLibAO/Abe.cpp

```diff
@@ -1125,6 +1125,9 @@ void Abe::vUpdate_41FDB0()
                     field_112_prev_motion = 0;
                     field_108_bMotionChanged = 1;
                     field_106_shot = 0;
+#if ORIGINAL_GAME_FIXES || ORIGINAL_GAME_FIX_DEATH_DELAY
+                    field_2A8_flags.Clear(Flags_2A8::e2A8_Bit2); // OG Change - Fixes "Death Delay Glitch"
```

Comment:
Is the meaning of known? If so can we rename it whatever it actually means? Also does this bug not happen in AE?

Reply:
Do you mean the meaning of "e2A8_Bit2"? 
It seems to be the equivalent of "e1AC_Bit2_return_to_previous_motion" in AE.
Would you like me to rename all instances of "e2A8_Bit2" to "e2A8_Bit2_return_to_previous_motion"?
The bug can happen in AE: https://www.youtube.com/watch?v=GXlXGiCgI_M
But I'm not sure how it is triggered... Might be with the electric fences.

Reply:
yes renaming to e2A8_Bit2_return_to_previous_motion would be good, since it can happen in AE but isn't fixed there you should change ORIGINAL_GAME_FIX_DEATH_DELAY to be ORIGINAL_GAME_FIX_DEATH_DELAY_AO too for now